apps/bulldozer-client/src/styles/components/custom-containers.scss

```diff
@@ -0,0 +1,38 @@
+@use 'sass:map';
+@use '@angular/material' as mat;
+@import '../constants/sizes.scss';
+
+// mixings to define color
+@mixin color($theme) {
+  // Get the color config from the theme.
+  $color-config: mat.get-color-config($theme);
+
+  // Get the primary color palette from the color-config.
+  $primary-palette: map.get($color-config, 'background');
+
+  .bd-custom-height-sidebar {
+    #bd-application-buttons {
+      background-color: mat.get-color-from-palette(
+        $primary-palette,
+        'background'
+      );
+    }
+  }
+}
+
+@mixin theme($theme) {
+  $color-config: mat.get-color-config($theme);
+  @if $color-config != null {
+    @include color($theme);
+  }
+}
+
+.bd-custom-height-sidebar {
+  #bd-application-panel {
+    overflow-y: auto;
+    height: calc(100vh - $toolbar-height - $actions-buttons-application);
+  }
+  #bd-application-buttons {
+    height: $actions-buttons-application;
```

Comment:
why create a new style rather than using a tailwind class for the application buttons?